src/lib.rs

```diff
@@ -1,436 +1,11 @@
-// Copyright ⓒ 2015-2016 Kevin B. Knapp and [`clap-rs` contributors](https://github.com/kbknapp/clap-rs/blob/master/CONTRIBUTORS.md).
+// Copyright ⓒ 2015-2016 Kevin B. Knapp and [`clap-rs` contributors](https://github.com/clap-rs/clap/graphs/contributors).
```

Comment:
While we're at it, might as well update the copyright year too :stuck_out_tongue_winking_eye: